docs/source/basic/install.rst

```diff
@@ -3,44 +3,61 @@
 Installation
 ============
 
-* Clone alpaka from github.com
-
 .. code-block::
 
+  # Clone alpaka from github.com
   git clone https://github.com/alpaka-group/alpaka
```

Comment:
Maybe we should recommend using the stable branch here?

Reply:
`git clone --branch 0.8.0 https://github.com/alpaka-group/alpaka.git`

Reply:
Yes, that is what I meant.